.github/ISSUE_TEMPLATE/release.md

```diff
@@ -20,4 +23,6 @@ Steps:
 
 The full process is described at https://github.com/openfoodfoundation/openfoodnetwork/wiki/Releasing.
 
+[auto update]: https://www.transifex.com/open-food-foundation/open-food-network/content/
```

Comment:
Where can I see that? where is it supposed to be? I guess it's already disabled

Reply:
If you go to that URL, there is a button at the top right (maybe you need to be logged in) to configure the auto-update. It was disabled, yes. But Luis' process was to enable it after the release has been completed and disable it again when starting to draft the release. So while some people may use that process, we need to double-check this here.